Classes/Expression.lua

```diff
@@ -0,0 +1,253 @@
+
+-- Expression.lua
+
+-- Contains the cExpression class. This allows formulas to be executed safely in an empty environment.
+--[[ Usage example:
+local FormulaString = "data=4; x > y"
+local Expression = cExpression:new(FormulaString)
+
+Expression:AddReturnValue("Comp1")           -- Return the first known comparisons
```

Comment:
Wait, the comparison results can be returned? Why is this not mentioned elsewhere? I'd expect it in the `AddReturnValue` docs and `Compile` docs.